packages/familie-vedlegg/README.md

```diff
@@ -0,0 +1,13 @@
+# Visittkort
+
+Komponent for å vise dokumentvedlegg for team familie.
```

Comment:
Hva er et dokumentvedlegg?

Reply:
PDF-vedlegg er vel kanskje mer presist. Men ikke XML/JSON-vedlegg. 
Alternativt bare "vedlegg"

Målet med pakken : felleskomponent for vedleggsvisning i saksbehandlingsløsningen?

Reply:
Men er det for visning av vedlegg eller dokumenter/journalposter? Kan man ikke fylle listen med annet enn vedlegg (altså ikke selve søknaden, bare vedleggene?)

Reply:
Dette er, tenkt i EF, en opplisting av metadataene til vedleggene/dokumentene fra journalposter.
Vi viser ikke selve vedleggene. De må lastes ned - det blir en funksjon som sendes med til komponenten. 
Selve søknaden (pdf-versjonen) er også et "vedlegg" i denne konteksten.

Men hvorvidt det bør hete "vedlegg", "dokumenter" eller noe helt annet er jeg ikke sikker på... Forslag?
Jeg er korka etter karantene siden fredag og to dager med hjemmeskole+hjemmekontor, så vet ikke om jeg kommer på et bedre begrep nå...

Reply:
hmm, men hvert "vedlegg" i listen peker jo til én journalpostId. Så da ville jeg antatt at et "vedlegg" er en journalpost og at vedlegget viser info om journalpost og alle dokumenter på journalposten? 🤷